variables.tf

```diff
@@ -31,6 +31,6 @@ variable "fpjs_shared_secret" {
 
 variable "fpjs_proxy_lambda_role_permissions_boundary_arn" {
   type        = string
-  description = "permissions boundary ARN for the role assumed by the Proxy lambda"
+  description = "Permissions boundary ARN for the role assumed by the Proxy lambda. Make sure your permissions boundary allows the function to access the Secret manager secret created for the integration (`secretsmanager:GetSecretValue`) and create logs (`logs:CreateLogStream`, `logs:CreateLogGroup`, `logs:PutLogEvents`)."
```

Comment:
Could you rename "Secret manager" > "Secrets Manager" please? With s and with capital M. @JuroUhlar

Reply:
Done thanks: https://github.com/fingerprintjs/terraform-aws-fingerprint-cloudfront-proxy-integration/pull/34/commits/ac3e186e0846b768724aaa1bbc63d2e580cba84c